test/services/process_ping_things_service_test.rb

```diff
@@ -21,6 +21,28 @@ class ProcessPingThingsServiceTest < ActiveSupport::TestCase
     assert 10, PingThing.count
   end
 
+  test 'service \
+    with correct input \
```

Comment:
Wrong indentation. It's also really weird unintuitive format, especially for such a short line.